CONTRIBUTING.md

```diff
@@ -1,6 +1,25 @@
 # Contributing to Salt
 
-To do: intro.
+Follow below instructions to contribute to Salt.
+
+1. Run `yarn` to install dependencies. If this step stuck, check your cooperate proxy setting.
+1. Run `yarn build` to build all packages across the repo.
```

Comment:
```suggestion
1. Run `yarn build` to build all packages across the repo. This is required to run Storybook or the site.
```
Maybe this will prevent people skipping it. Might not be needed though